### PR TITLE
Saas 11.5 fix vsc

### PR DESCRIPTION
**Pull request by VincentSchippefilt**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/js/libs/underscore.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/**
```
Comment (mge-odoo) [inline addons/web/static/src/js/libs/underscore.js]: I would put it in a `odoo.define('web.underscore.extensions', function () {` like other extensions.

File: addons/web/static/src/js/libs/underscore.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+ * the _throttle in underscore has the feature to cancelled the throttled
```
Comment (mge-odoo) [inline addons/web/static/src/js/libs/underscore.js]: **T**he

Comment (mge-odoo) [reply addons/web/static/src/js/libs/underscore.js]: `throttle`

Comment (mge-odoo) [reply addons/web/static/src/js/libs/underscore.js]: cancel

File: addons/web/static/src/js/libs/underscore.js (hunk 1 above)
Comment (mge-odoo) [reply addons/web/static/src/js/libs/underscore.js]: Missing `'use strict';`

File: addons/web/static/src/js/libs/underscore.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    };
+
+    throttled.cancel = function () {
+        clearTimeout(timeout);
+        previous = 0;
+        timeout = context = args = null;
+    };
+
+    return throttled;
+};
```
Comment (mge-odoo) [inline addons/web/static/src/js/libs/underscore.js]: missing line break

Comment (mge-odoo) [reply addons/web/static/src/js/libs/underscore.js]: I didn't check the function core as I suppose that you copy pasted it from undescore.

File: addons/account/views/report_invoice.xml [hunk 4]
```diff
@@ -19,28 +19,28 @@
                         <span t-field="o.number"/>
                     </h2>
 
-                    <div class="row col mt32 mb32">
-                        <div class="mr32" t-if="o.name" name="description">
+                    <div class="row mt32 mb32">
+                        <div class="mr32 col-3" t-if="o.name" name="description">
```
Comment (mge-odoo) [inline addons/account/views/report_invoice.xml]: maybe just use `col` as discussed, to avoid line break?
are all mr/mb still useful?

Comment (VincentSchippefilt) [reply addons/account/views/report_invoice.xml]: actually it will be less nice if there are a small number of elements

**Pull request merged**